bgpd/bgp_route.c

```diff
@@ -10489,9 +10526,14 @@ DEFUN (show_ip_bgp,
 		return bgp_show_filter_list(vty, bgp, argv[idx + 1]->arg, afi,
 					    safi, bgp_show_type_filter_list);
 
-	if (argv_find(argv, argc, "statistics", &idx))
-		return bgp_table_stats(vty, bgp, afi, safi);
-
+	if (argv_find(argv, argc, "statistics", &idx)) {
+		if (uj)
+			json = json_object_new_object();
+		ret = bgp_table_stats(vty, bgp, afi, safi, json);
```

Comment:
For loop over printing a single json object will not produce valid json.

Did you run any of this output through a json parser?

Reply:
I use now array functionality.